File: java/decorador/decorador/src/main/java/edu/uea/MilkDecorator.java
package edu.uea;

// Decoradores concretos
class MilkDecorator extends BeverageDecorator {
    public MilkDecorator(Beverage beverage) {
        super(beverage);
    }

    public String getDescription() {
        return beverage.getDescription() + ", Milk";
    }

    public double cost() {
        return beverage.cost() + 1.5;
    }
}

File: python/decorador/MilkDecorator.py
from BeverageDecorator import BeverageDecorator

# Decoradores concretos
class MilkDecorator(BeverageDecorator):
    def get_description(self):
        return self.beverage.get_description() + ", Milk"

    def cost(self):
        return self.beverage.cost() + 1.5